src/components/AlefSymbol.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import styled from 'styled-components'
+
+import alefSymbol from '../images/alef.svg'
+
+interface AlefSymbolProps {
+  color?: string
+  className?: string
+}
+
+const AlefSymbol = ({ className, color }: AlefSymbolProps) => {
+  return (
```

Comment:
Maybe to save you and future us some pain we can use `useTheme` here to get the color? Then the symbol can color itself.

Reply:
I don't see how using this hook can help me customize the color of the symbol based on the color of the text where it is used at. I need to set it to red when used in error messages, gray when used in input field labels, and green when used in confirmed transaction details.